catroid/src/main/java/org/catrobat/catroid/ui/ProjectUtils.kt

```diff
@@ -57,14 +59,17 @@ import org.catrobat.catroid.content.bricks.WebRequestBrick
  * extension boolean function for List<Brick> data type.
  * check if the list contains suspicious bricks
  * */
-private fun List<Brick>.containsSuspiciousBricks(): Boolean {
+private fun List<Brick>.containsSuspiciousBricks(): Pair<Boolean, Boolean> {
```

Comment:
Please don't use a Pair for something like that.

Either iterate for every property separately, or at least use a Class / POJO to hold the data.